docs/source/guide/ddd-5-theory.myst

```diff
@@ -4,12 +4,91 @@ jupytext:
+
+In the context of quantum computing, DD can be considered as an error mitigation method.
+With respect to other error mitigation techniques, DDD has very peculiar features:
+
+- It maps a noisy quantum computation to a _single_ error-mitigated computation (no need to take linear combinations
+of noisy results as in [ZNE](zne-5-theory.myst), [PEC](pec-5-theory.myst) and [CDR](cdr-5-theory.myst)).
+
+- As a consequence of the previous point, there is not a fundamental error mitigation overhead or
+an increase of statistical uncertainty in the final results.
+
+- If noise is time-correlated, it can suppress real errors at the physical level instead of applying a virtual noise
+reduction via classical post-processing.
+
+
+
+
+## Digital dynamical decoupling
+
+In a quantum computing device based on the circuit model, sequences of DD pulses can be mapped to sequences
+of discrete quantum gates (typically Pauli gates). We refer to this gate-level formulation as _digital dynamical decoupling_ (DDD)
+to distinguish it from the standard pulse-level formulation.
+
+
+
+```{note}
+This type of gate-level approach is very similar to the gate-level abstraction used in Mitiq to implement
+_digital zero-noise extrapolation_ via _unitary folding_ (see [What is the theory behind ZNE?](zne-5-theory.myst)).
+```
+Experimental evidence showing the practical utility gate-level decoupling sequences is given in several publications {cite}`Pokharel_2018_PRL, Jurcevic_2021_arxiv, GoogleQuantum_2021_nature, Smith_2021_arxiv, Das_2021_ACM`.
+
+
+```{warning}
+Gate-level DDD can only be considered as an approximation of the ideal (pulse-level) DD technique. Moreover, quantum backends 
+may internally optimize and schedule gates in unpredictable ways such that, in practice, DDD sequences may not be physically applied
+as expected.
+```
+
+A significant advantage of DDD with respect to pulse-level DD is the possibility of applying it in a backend-independent way, 
+via simple transformations of abstract quantum circuits. For this reason, DDD is particularly suitable for a multi-platform library like Mitiq.
+
+
+
+## Common examples of DDD sequences
+
+Common dynamical decoupling sequences are arrays of (evenly spaced) Pauli gates. In particular:
+- The _XX_ sequence is typically appropriate for mitigating (time-correlated) dephasing noise;
+- The _YY_ sequence is typically appropriate for mitigating (time-correlated) amplitude damping noise;
+- The _XYXY_ sequence is typically appropriate for mitigating generic single-qubit noise.
+
+All the above examples of DDD sequences are supported in Mitiq and more general ones can be defined and customized by users.
+For more details on how to define DDD sequences in Mitiq see [What additional options are available for DDD?](ddd-3-options.myst). 
+
+```{note}
+A general property of DDD sequences is that, if executed on a noiseless backend, they are equivalent to the identity operation.
+```
```

Comment:
This feels very important, and maybe like it should occur higher up in the article? Not sure where best to put it though.

Reply:
Moved up by 1 paragraph. I wasn't able to do more without risking to confuse the reader.